tests/functional/olp-cpp-sdk-dataservice-read/DataserviceReadVolatileLayerClientTest.cpp

```diff
@@ -46,6 +51,19 @@ const auto kAppSecretEnvName = "dataservice_read_volatile_test_secret";
 const auto kCatalogEnvName = "dataservice_read_volatile_test_catalog";
 const auto kLayerEnvName = "dataservice_read_volatile_layer";
 
+const auto kPrefetchAppId = "dataservice_read_volatile_test_prefetch_appid";
```

Comment:
could we have one app id and secret for all tests in VolatileLayerClientTest? So other user can set only once their id/secret.

Reply:
Will need to change old tests, so they use new catalog, layers etc.
Old credentials/catalog can't be used in new tests, since need to write data with tiles first.

Anyway, there is a separated task to create accessible catalogs for all tests. After that we could use same credentials for all tests here.